gateway/src/main/java/io/zeebe/gateway/impl/broker/BrokerClientImpl.java

```diff
@@ -193,16 +209,16 @@ public void close() {
 
   @Override
   public <T> ActorFuture<BrokerResponse<T>> sendRequest(
-      BrokerRequest<T> request, Duration requestTimeout) {
+      final BrokerRequest<T> request, final Duration requestTimeout) {
     return requestManager.sendRequest(request, requestTimeout);
   }
 
   @Override
   public <T> void sendRequest(
-      BrokerRequest<T> request,
-      BrokerResponseConsumer<T> responseConsumer,
-      Consumer<Throwable> throwableConsumer,
-      Duration requestTimeout) {
+      final BrokerRequest<T> request,
```

Comment:
Just out of curiosity, why are these variables now decorated as final given that once this method goes out of scope, these variables are candidate for GC.

Reply:
I usually prefix method arguments with final to discourage reassigning parameters in a method body. In my experience this can lead to confusing code. So I have my IDE set up to automatically prefix arguments with it.

In this case it (i.e. a constructor) it is probably not necessary as we shouldn't be doing much work in a constructor anyway, but like I said my IDE just adds it to everything. This is just personal preference though, I'd be glad to hear if you have any thoughts against it